packages/story-editor/src/app/story/useStoryReducer/test/addGroup.js

```diff
@@ -79,6 +79,26 @@ describe('addGroup', () => {
     });
   });
 
+  it('should add group even if there are no groups', () => {
```

Comment:
Nit:
Is this for testing it works when `groups` is not set yet? Since in the previous test there are also no groups but the `groups` prop is set, just empty. Perhaps this then:
```suggestion
  it('should add group even if `groups` is not set, () => {
```

Reply:
True, the text could be more explicit. Or the tests more . Then empty `groups` object is a bit rare, so either it should be there and be non-empty or non be there at all.